app/components/applications-app-services/function-app.tf

```
@@ -218,6 +218,50 @@ resource "azurerm_role_assignment" "nsip_representation_service_bus_role" {
   principal_id         = module.back_office_subscribers[0].principal_id
 }
 
+resource "azurerm_servicebus_subscription_rule" "nsip_representation_topic_subscription_rule" {
+  count = var.feature_back_office_subscriber_enabled ? 1 : 0
+
+  name            = "applications-nsip-representation-subscription-rule"
+  subscription_id = azurerm_servicebus_subscription.nsip_representation_topic_subscription[0].id
+  filter_type     = "CorrelationFilter"
+  correlation_filter {
+    properties = {
+      type = "Publish"
+    }
+  }
+}
+
+# nsip-representation - UPDATE
+
+resource "azurerm_servicebus_subscription" "nsip_update_representation_topic_subscription" {
+  count = var.feature_back_office_subscriber_enabled ? 1 : 0
+
+  name               = "applications-nsip-representation-update"
+  topic_id           = var.back_office_service_bus_nsip_representation_topic_id
+  max_delivery_count = 1
+}
+
+resource "azurerm_role_assignment" "nsip_representation_service_bus_role" {

```

```suggestion
resource "azurerm_role_assignment" "nsip_representation_update_service_bus_role" {

```

validate is complaining about this resource as the name has already been used by a resource of the same type (line 213)